editor/editor_node.cpp

```diff
@@ -2066,8 +2066,9 @@ void EditorNode::_dialog_action(String p_file) {
 		} break;
 
 		case FILE_EXPORT_MESH_LIBRARY: {
-			bool merge_with_existing_library = file_export_lib_merge->is_pressed();
-			bool apply_mesh_instance_transforms = file_export_lib_apply_xforms->is_pressed();
+			const Dictionary &fd_options = file_export_lib->get_selected_options();
+			bool merge_with_existing_library = (bool)fd_options.get(TTR("Merge With Existing"), true);
+			bool apply_mesh_instance_transforms = (bool)fd_options.get(TTR("Apply MeshInstance Transforms"), false);
```

Comment:
```suggestion
			bool merge_with_existing_library = fd_options.get(TTR("Merge With Existing"), true);
			bool apply_mesh_instance_transforms = d_options.get(TTR("Apply MeshInstance Transforms"), false);
```
Explicit cast is not required when assigning.